### ISSUE TITLE
Support for Microsoft IE 11 Browser

### ISSUE THREAD

**Issue by mahmoudzamel** [DominoKit/domino-ui#115]
We'd like that the UI kit provides support for Microsoft IE 11 Browser; as our customer in hand has so many client workstations that are policy restricted to IE 11.
It would be very kind of you if you started working on it the soonest; as we are targeting Go Live by October 1st.

**Comment by vegegoku** [comment]
Mainly we have few issues with IE11, we know about `element.remove()` not being supported, Material Icons not working, And css-grids does not support auto placement.

We already found the solution for the Icons issue, the remove issue sounds like and easy fix.
css-grid we can provide a specific implementation for IE11 that only supports 12 columns grid which should cover 90% of the use cases. this grid system support might take few days to be implemented if we got to work full time on this.
